packages/contextual-menu-react/src/ContextualMenu.test.tsx

```diff
@@ -101,6 +139,32 @@ describe("ContextualMenu", () => {
         expect(ekspandert).toBeInTheDocument();
     });
 
+    test("should call onToggle callback when opening and closing", async () => {
+        const onToggle = jest.fn();
+
+        const { getByRole, user } = setup(
+            <ContextualMenu
+                onToggle={onToggle}
+                initialPlacement="bottom-start"
+                triggerElement={
+                    <IconButton title="En kontekstuell meny">
+                        <DotsIcon bold />
+                    </IconButton>
+                }
+            >
+                <ContextualMenuItem>Menyvalg</ContextualMenuItem>
+            </ContextualMenu>,
+        );
+
+        await user.click(getByRole("button", { name: "En kontekstuell meny" }));
+        await user.click(getByRole("button", { name: "En kontekstuell meny" }));
+
+        // Called with false on mount since menu starts out closed
```

Comment:
Litt usikker på om dette er villet oppførsel, men dersom man feks bruker `isOpen` prop for å kontrollere state så kan det være greit at komponenten melder fra om at den faktisk er åpen. 

Alternativt kan man holde noe intern state som sørger for at callback ikke kalles på første render (mount)